src/provisioner/provisioner.py

```diff
@@ -96,11 +96,17 @@ def cluster_bootstrap(node_id: str, cortx_conf_url: str = None):
 
         components = cortx_config_store.get(f'node>{node_id}>components')
         num_components = len(components)
-        for j in range(0, num_components):
-            services = cortx_config_store.get(f'node>{node_id}>components[{j}]>services')
-            service = 'all' if services is None else ','.join(services)
-            print(f"{components[j]['name']}_setup --config {CortxProvisioner._cortx_conf_url} --services %s" %service)
-
-            # TODO: Enable this code
-            # rc, output = SimpleProcess(f"{components[i]}_setup --config \
-            #  {CortxProvisioner._cortx_conf_url} --services %s" %services.join(","))
+        mp_interfaces = ['post_install', 'prepare', 'config', 'init']
+        for interface in mp_interfaces:
+            for j in range(0, num_components):
```

Comment:
Orchestration goes like this
1. comp1:post_install
2. comp2: post_install
3. comp1:prepare
4. comp2:prepare..
5. ...

Reply:
yes @ujjwalpl 

Below is the flow
```
motr_setup --config yaml:///tmp/services.yaml post_install --services io
hare_setup --config yaml:///tmp/services.yaml post_install --services all
s3_setup --config yaml:///tmp/services.yaml post_install --services io,auth,bg_consumer
utils_setup --config yaml:///tmp/services.yaml post_install --services message_bus
motr_setup --config yaml:///tmp/services.yaml prepare --services io
hare_setup --config yaml:///tmp/services.yaml prepare --services all
s3_setup --config yaml:///tmp/services.yaml prepare --services io,auth,bg_consumer
utils_setup --config yaml:///tmp/services.yaml prepare --services message_bus
motr_setup --config yaml:///tmp/services.yaml config --services io
hare_setup --config yaml:///tmp/services.yaml config --services all
s3_setup --config yaml:///tmp/services.yaml config --services io,auth,bg_consumer
utils_setup --config yaml:///tmp/services.yaml config --services message_bus
motr_setup --config yaml:///tmp/services.yaml init --services io
hare_setup --config yaml:///tmp/services.yaml init --services all
s3_setup --config yaml:///tmp/services.yaml init --services io,auth,bg_consumer
utils_setup --config yaml:///tmp/services.yaml init --services message_bus
```